app/services/foreman/parameters/validator.rb

```diff
@@ -13,6 +13,10 @@ def validate!
           validate_regexp
         when "list", "array"
           validate_list
+        when "hash"
```

Comment:
it's actually a regression of sorts, because now for some reason the value gets turned into yaml and breaks even more...

something changed in develop an now this exception is no longer raised, but it's acting worse.